apps/pie-docs/src/assets/styles/components/_nav.scss

```diff
@@ -71,18 +80,17 @@ $nav-item-active-left-padding:        calc($nav-item-left-padding - $nav-item-ac
 
 .c-nav-item {
   display: block;
-  padding: f.spacing(d) f.spacing(d) f.spacing(d) $nav-item-left-padding;
+  padding: 10px f.spacing(d);
```

Comment:
Same as @jamieomaguire mentioned above for this spacing value

Reply:
we checked with @jamieomaguire and agreed to keep the 10px value as it's quite specific and adheres to our element-first approach.